### DISCUSSION TITLE
Enquiry

### DISCUSSION THREAD

**Discussion by Tammy-Ajoko** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#40] [Outreachy 2022] [answered]
@BatoolMM I created an issue could please look over it.
https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/183

**Comment by BatoolMM** [comment, marked as answer]
hi @Tammy-Ajoko, 

We will review your PR as soon as we can!
